pkg/crc/segment/segment_test.go

```diff
@@ -0,0 +1,55 @@
+package segment
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	crcConfig "github.com/code-ready/crc/pkg/crc/config"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	ConsentTelemetry = "consent-telemetry"
+)
+
+func newTestConfig(configFile string, value bool, envPrefix string) (*crcConfig.Config, error) {
```

Comment:
To avoid the file, you can do the same as https://github.com/code-ready/crc/blob/56356140518b158b1e6119e0f9e9598d85d2db17/pkg/crc/api/api_test.go#L200